usecases/traverser/metrics.go

```diff
@@ -116,20 +112,3 @@ func (m *Metrics) QueriesGetDec(className string) {
 		"query_type": "get_graphql",
 	}).Dec()
 }
-
-func (m *Metrics) AddUsageDimensions(className, queryType, operation string, dims int) {
```

Comment:
this is only used in `explorer` so moved it from `traverser` -> `explorer` method.